### ISSUE TITLE
Need list of all available tooling (separate from "How to" instructions)?

### ISSUE THREAD

**Issue by olivekl** [slsa-framework/slsa#697]
As SLSA tooling increases, it'd be good to have one place that captures all available tools across ecosystems and levels. Currently some information is available in the "How to SLSA"/"Getting started" instructions, where tools like the slsa generator and provenance verifier are mentioned as part of the instructions. To get a full picture of what's available, though, someone would have to read multiple blog post announcements and visit multiple repos.

I think we need a centralized list that can be added to as this area grows, perhaps in table format that shows who should use what tool according to their use case (i.e., should I choose the generic provenance generator or the Go-specific one, and why?). 

@ianlewis @asraa @laurentsimon -- if I start this doc, would you be able to help populate information  and guidance about use cases?

**Comment by laurentsimon** [comment]
Yes @olivekl we'd be happy to help. We made the same observation.

**Comment by mlieberman85** [comment]
Closing as inactive — no activity since this was filed. If it's still worth looking at, feel free to reopen or file a new ticket with updated context.
